### PR TITLE
Add static files serving example

### PR DESCRIPTION
**Pull request by simon-johansson**

### CODE REVIEW

File: ts/serve-static-files/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "type": "module",
+  "scripts": {
+    "test": "vitest"
+  },
+  "devDependencies": {
+    "@types/node": "^20.5.7",
+    "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "encore.dev": "^1.40.5"
```
Comment (eandre) [inline ts/serve-static-files/package.json:16]: Should make this v1.41.1 to include the SDK for static files
```suggestion
    "encore.dev": "^1.41.1"
```

**Comment by simon-johansson** [comment]
Thank for the fixes @eandre 💪

File: README.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ environment. Use the appropriate command for your system:
 | [ts/sequelize](ts/sequelize)                                     | Encore + Sequelize TypeScript Example                                                                                    | APIs, SQL Database                                                     |
 | [ts/expressjs-migration](ts/expressjs-migration)                 | Express.js migration guide examples                                                                                      | APIs, Raw Endpoints, Auth, Databases                                   |
 | [ts/file-upload](ts/file-upload)                                 | Upload files from frontend example                                                                                       | Raw Endpoints                                   |
+| [ts/serve-static-files](ts/serve-static-files)                   | Serving static files example                                                                                             | Raw Endpoints                                   |
```
Comment (eandre) [inline README.md:69]: Maybe we should simplify the example name to just `ts/static-files`?

Comment (simon-johansson) [reply README.md]: Done

**Review by eandre — APPROVED** [review]

**Pull request merged**